integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -35,6 +35,11 @@ public class ArgumentParser {
       "spark.openlineage.parentJobNamespace";
   public static final String SPARK_CONF_PARENT_JOB_NAME = "spark.openlineage.parentJobName";
   public static final String SPARK_CONF_PARENT_RUN_ID = "spark.openlineage.parentRunId";
+  public static final String SPARK_CONF_ROOT_PARENT_JOB_NAMESPACE =
```

Comment:
Could you update Spark doc in the same PR?

Reply:
Done.